cs41_final.sh

```diff
@@ -74,59 +176,77 @@ while [ userChoice != '0' ]
       case $userChoice in
         1) clear;
         option_picked "Option 1: List of current Users";
-        #do something
+        currentUsers
         show_menu;
         ;;
 
         2) clear;
         option_picked "Option 2: List of active processes for user ${userName}";
-        #do something
-        show_menu;
+        echo "Currently logged-in users:";
+        currentUsers;
+        echo "Please select from available list.";
+        read userName;
+        #do something: check for validity, if not valid, prompt again. when true do something
+        listActiveProcess;
+        show_menu;       show_menu;
         ;;
 
         3) clear;
-        option_picked "Option 3: Active processes for user ${userName}";
-        #do something;
+        option_picked "Option 3: Monitor active processes for user ${userName}";
+        echo "Currently logged-in users:";
+        currentUsers;
+        echo "Please select from available list.";
+        read userName;
```

Comment:
this currently stalls the whole program.  this will probably get fixed once the functions are done :)